src/pyhf/infer/calculators.py

```diff
@@ -155,14 +155,21 @@ def teststatistic(self, poi_test):
 
         """
         tensorlib, _ = get_backend()
-        qmu_v = qmu(poi_test, self.data, self.pdf, self.init_pars, self.par_bounds)
+
+        teststat_func = qmu_tilde if self.qtilde else qmu
+
+        qmu_v = teststat_func(
```

Comment:
we probably should not call it `qmu_v` but instead `test_statistic_v` or similar.

Reply:
@kratsg Actually, do we want to change that now? At the moment we're only importing

```
from .test_statistics import qmu, qmu_tilde
```

so we're not really intending there to be direct `tmu` and `tmu_tilde` uses at the moment until we implement more support for two sided intervals. @lukasheinrich do I have that right?

Though maybe we do want to rename some things still to make it more clear that this can be `qmu` or `qmu_tilde` releated.